server/src/main/java/org/elasticsearch/inference/UnifiedCompletionRequest.java

```diff
@@ -143,7 +148,7 @@ private static Content parseContent(XContentParser parser) throws IOException {
 
         public Message(StreamInput in) throws IOException {
             this(
-                in.readNamedWriteable(Content.class),
+                in.readOptionalNamedWriteable(Content.class),
```

Comment:
Just noting here we don't need to change the transport version because this code is feature flagged